sklearn/decomposition/tests/test_nmf.py

```diff
@@ -154,14 +244,30 @@ def test_nmf_transform(solver):
         n_components=3,
         init="random",
         random_state=0,
-        tol=1e-5,
+        tol=1e-6,
     )
     ft = m.fit_transform(A)
     t = m.transform(A)
-    assert_array_almost_equal(ft, t, decimal=2)
+    assert_allclose(ft, t, atol=1e-1)
```

Comment:
If the implementation of `NMF` has not changed, why are those changes for the tolerances needed?